src/spring/azext_spring/_validators.py

```diff
@@ -106,6 +106,8 @@ def _check_tanzu_components_not_enable(cmd, namespace):
         raise ArgumentUsageError('--enable-application-live-view {}'.format(suffix))
     if namespace.enable_application_accelerator:
         raise ArgumentUsageError('--enable-application-accelerator {}'.format(suffix))
+    if namespace.application_configuration_service_generation:
+        raise ArgumentUsageError('--application_configuration_service_generation {}'.format(suffix))
```

Comment:
```suggestion
        raise ArgumentUsageError('--application-configuration-service-generation {}'.format(suffix))
```

Reply:
Besides, it is suggested to point out the error in more details. It is better to specify how to input.